ivy/functional/ivy/data_type.py

```diff
@@ -1861,7 +1861,27 @@ def unset_default_dtype():
 
 @handle_exceptions
 def unset_default_float_dtype():
-    """"""
+    """Reset the current default float dtype to the previous state.
+
+    Examples
+    --------
+    >>> ivy.set_default_float_dtype(ivy.float64)
+    >>> ivy.default_float_dtype()
+    'float64'
+    >>> ivy.unset_default_float_dtype()
+    >>> ivy.default_float_dtype()
+    'float32'
```

Comment:
I don't think this example is useful. You can remove it and keep only the other one.

Reply:
alright, I have removed it